index.html

```diff
@@ -65,6 +65,204 @@ <h2 class="sub-heading">I'm a Software Developer</h2>
+        <ul class="card-technologies">
+          <li class="card-tech first-tech">html</li>
+          <li class="card-tech">bootstrap</li>
+          <li class="card-tech">Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="btn2">See Project</button>
+    </article>
+
+    <article class="card">
+      <div class="card-2">
+        <h3>Profesional Art Printing Data</h3>
+        <p>A daily selection of privately personalized reads; 
+         no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-technologies">
+          <li class="card-tech first-tech">html</li>
+          <li class="card-tech">bootstrap</li>
+          <li class="card-tech">Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="btn2">See Project</button>
+    </article>
+
+    <article class="card">
+      <div class="card-2">
+        <h3>Profesional Art Printing Data</h3>
+        <p>A daily selection of privately personalized reads; 
+         no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-technologies">
+          <li class="card-tech first-tech">html</li>
+          <li class="card-tech">bootstrap</li>
+          <li class="card-tech">Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="btn2">See Project</button>
+    </article>
+
+    <article class="card">
+      <div class="card-2">
+        <h3>Profesional Art Printing Data</h3>
+        <p>A daily selection of privately personalized reads; 
+         no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-technologies">
+          <li class="card-tech first-tech">html</li>
+          <li class="card-tech">bootstrap</li>
+          <li class="card-tech">Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="btn2">See Project</button>
+    </article>
+
+    <article class="card">
+      <div class="card-2">
+        <h3>Profesional Art Printing Data</h3>
+        <p>A daily selection of privately personalized reads; 
+         no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-technologies">
+          <li class="card-tech first-tech">html</li>
+          <li class="card-tech">bootstrap</li>
+          <li class="card-tech">Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="btn2">See Project</button>
+    </article>
+  </section>
+
+  <section class="about-me" id="about">
+    <div class="about">
+      <h2>About me</h2>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website.
+        Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hesitate to contact me.
+      </p>
+      <button class="btn3" type="submit">Get My Resume</button>
+      <hr class="hr2">
+      <div class="exp">
+        <div class="language">
```

Comment:
- [ ] Kindly use the appropriate background color for your about section as required by the previous reviews  I suggest you use this one background: #1C1A19; it will make your design stick to the Figma design :+1: 
- [ ] Kindy also apply the appropriate font family for Languages, Frameworks, and Skills as required by the previous reviewer it will make your design stick more to the Figma design :+1: